io_export_paper_model.py

```diff
@@ -1306,52 +1330,61 @@ def __init__(self, uvedge, default_width, index, other: UVEdge):
             other_edge_neighbor_a = other_face_neighbor_left.vb.co - other.vb.co
             other_edge_neighbor_b = other_face_neighbor_right.va.co - other.va.co
             # Adjacent angles in the face
-            cos_a = max(cos_a, -other_edge.dot(other_edge_neighbor_a) / (other_edge.length*other_edge_neighbor_a.length))
-            cos_b = max(cos_b, other_edge.dot(other_edge_neighbor_b) / (other_edge.length*other_edge_neighbor_b.length))
+            cos_a = max(cos_a,
+                        -other_edge.dot(other_edge_neighbor_a) / (other_edge.length * other_edge_neighbor_a.length))
+            cos_b = max(cos_b,
+                        other_edge.dot(other_edge_neighbor_b) / (other_edge.length * other_edge_neighbor_b.length))
         except AttributeError:  # neighbor data may be missing for edges with 3+ faces
             pass
         except ZeroDivisionError:
             pass
 
         # Calculate the lengths of the glue tab edges using the possibly smaller angles
-        sin_a = abs(1 - cos_a**2)**0.5
+        sin_a = abs(1 - cos_a ** 2) ** 0.5
         len_b = min(len_a, (edge.length * sin_a) / (sin_a * cos_b + sin_b * cos_a))
-        len_a = 0 if sin_a == 0 else min(sticker_width / sin_a, (edge.length - len_b*cos_b) / cos_a)
+        len_a = 0 if sin_a == 0 else min(sticker_width / sin_a, (edge.length - len_b * cos_b) / cos_a)
 
-        sin_b = abs(1 - cos_b**2)**0.5
+        sin_b = abs(1 - cos_b ** 2) ** 0.5
         len_a = min(len_a, (edge.length * sin_b) / (sin_a * cos_b + sin_b * cos_a))
         len_b = 0 if sin_b == 0 else min(sticker_width / sin_b, (edge.length - len_a * cos_a) / cos_b)
 
         tangent = edge.normalized()
         cos, sin = tangent
         self.rot = M.Matrix(((cos, -sin), (sin, cos)))
 
-        self.width = sticker_width * 0.9
+        self.width = sticker_width 
 
-        tab = svg2uv("C:\Program Files\\Blender Foundation\\Blender 2.81\\2.81\\scripts\\addons\\Stickers\\tooth-sawtooth.svg")
+        num_tabs = floor(edge.length/self.width)
+
+        tab = svg2uv(
+            "C:\Program Files\\Blender Foundation\\Blender 2.81\\2.81\\scripts\\addons\\Stickers\\tooth-sawtooth.svg")
 
         tab_verts = []
         tab_verts_co = []
```

Comment:
list comprehension might be changing state of the class, possible "getCo" fun to be implemented???